### PR TITLE
Landing  page - Terms and conditions and Privacy links not visible

### PR DESCRIPTION
**Pull request by Fbasham**
## [ADO-342](https://dev.azure.com/JourneyLab/SeniorsJourney/_sprints/taskboard/DTS/SeniorsJourney/DTS/DTS%20-%20Sprint%205?workitem=342)

List of proposed changes:
- change styles to make links visible at smaller viewport widths

### CODE REVIEW

File: frontend/src/components/Footer.tsx [hunk 1]
```diff
@@ -133,10 +133,10 @@ const Footer: FC<FooterProps> = ({
             <h3 className="sr-only" id="footerNav">
               {footerNavHeader}
             </h3>
-            <div className="flex items-end justify-between md:items-center">
-              <ul className="hidden flex-col flex-wrap space-y-3 marker:text-xs md:list-inside md:list-disc md:flex-row md:items-center md:space-y-0 lg:flex">
+            <div className="flex items-end justify-between sm:items-center">
+              <ul className="flex flex-col gap-2 sm:flex-row marker:text-xs sm:list-inside sm:list-disc">
                 {links.map(({ link, linkText }) => (
-                  <li key={link} className="first:list-none md:mr-4">
+                  <li key={link} className="sm:first:list-none mr-4">
```
Comment (sebastien-comeau) [inline frontend/src/components/Footer.tsx:139]: Class sorting
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

```suggestion
                  <li key={link} className="mr-4 sm:first:list-none">
```

File: frontend/src/components/Footer.tsx [hunk 2]
```diff
@@ -133,10 +133,10 @@ const Footer: FC<FooterProps> = ({
             <h3 className="sr-only" id="footerNav">
               {footerNavHeader}
             </h3>
-            <div className="flex items-end justify-between md:items-center">
-              <ul className="hidden flex-col flex-wrap space-y-3 marker:text-xs md:list-inside md:list-disc md:flex-row md:items-center md:space-y-0 lg:flex">
+            <div className="flex items-end justify-between sm:items-center">
+              <ul className="flex flex-col gap-2 sm:flex-row marker:text-xs sm:list-inside sm:list-disc">
```
Comment (sebastien-comeau) [inline frontend/src/components/Footer.tsx:137]: Class sorting and remove gap on sm and up breakpoints.
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier
```suggestion
              <ul className="flex flex-col gap-3 marker:text-xs sm:list-inside sm:list-disc sm:flex-row sm:gap-0">
```

**Review by sebastien-comeau — COMMENTED** [review]
The `Go to top` footer link should be removed based on latest GCWeb version update. The footer logo class should be change to this `className="h-6 w-auto sm:h-10"`.

File: frontend/src/components/Footer.tsx (hunk 2 above)
Comment (Fbasham) [reply frontend/src/components/Footer.tsx]: I removed the mr-4 which negates the need to add sm:gap-0.  Footer link has been removed.  Footer logo styles changed to account for viewport.

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**